src/Trakx.CryptoCompare.ApiClient.Websocket/Model/HeartBeat.cs

```diff
@@ -1,10 +1,9 @@
 ﻿using System.Text.Json.Serialization;
 
-namespace Trakx.CryptoCompare.ApiClient.WebSocket.DTOs.Inbound
+namespace Trakx.CryptoCompare.ApiClient.Websocket.Model
 {
     public class HeartBeat : InboundMessageBase
     {
-        internal const string TypeValue = "999";
```

Comment:
Feels good to remove these I think, I didn't know if it was better to have them implemented in/near the class like that, or in a separate class only in charge of the mapping like you did...